README.md

```diff
@@ -1,23 +1,28 @@
-# kube-linter-action
+<p align="center"><img src="https://raw.githubusercontent.com/stackrox/kube-linter/main/images/logo/KubeLinter-horizontal.svg" width="360"></p>
 
-This is a GitHub action for scanning Kubernetes deployment files with [kube-linter](https://github.com/stackrox/kube-linter).  This includes both the action itself (.github/actions) and sample GitHub workflow (.github/workflows) and a test YAML.
+# kube-linter-action - KubeLinter GitHub Action
 
-Quick deployment:
+This is a GitHub action for scanning Kubernetes YAML files and Helm charts in your GitHub workflow with [kube-linter](https://github.com/stackrox/kube-linter).
 
-1.  Create a new GitHub repo.
-2.  Push all files from the `sample` directory into the repo.
-3.  The `kube-linter.yml` workflow will run as an action every time there's a new push to this repo.
+## Quickstart
 
-The action takes two parameters.
+1. Copy [.github/workflows/kube-linter-sample.yml](https://github.com/stackrox/kube-linter-action/tree/main/.github/workflows/kube-linter-sample.yml) file to `.github/workflows` directory in your repo.
+2. Adjust scan `directory` to the location where your Kubernetes or Helm files are. See Parameters below.
 
-```
-      - name: Scan repo
-        id: kube-lint-repo
-        uses: stackrox/kube-linter-action@v1
+The new workflow will run every time there's a new push to the repo.  
+Workflow will fail if kube-linter detects issues. You'll find issues in the output of `kube-linter-action`.
+
+### Example
+
+```yaml
+      - name: Scan repo with kube-linter
+        uses: stackrox/kube-linter-action@v1.0.2
         with:
           directory: yamls
           config: .kube-linter/config.yaml
 ```
 
-* `directory` is mandatory -- this is the directory of deployment files to scan.  
-* `config` is optional -- this is the path to a [configuration file](https://github.com/stackrox/kube-linter/blob/main/config.yaml.example) if you wish to use a non-default configuration.
+### Parameters
+
+* `directory` (required) - path of file or directory to scan, absolute or relative to the root of the repo.
+* `config` (optional) - path to a [configuration file](https://docs.kubelinter.io/#/configuring-kubelinter) if you wish to use a non-default configuration.
```

Comment:
Just a thought I had while reading this, it's not directly related to this PR.

We recently added support for auto detection of config files with well-known file names. (https://docs.kubelinter.io/#/configuring-kubelinter?id=configuring-kubelinter)

![image](https://user-images.githubusercontent.com/3502044/116746535-77413600-a9b1-11eb-9ad2-667827a74a94.png)

How does this intersect with GitHub actions? Is the working directory always the root of the repo? Should we say we will run `kube-linter` with the working directory equal to `directory` by default?

Reply:
> Is the working directory always the root of the repo?

Yes, it is repo root by default.

> Should we say we will run `kube-linter` with the working directory equal to `directory` by default?

We can. However `directory` can technically be a file path because `kube-linter` allows to scan a specific file. In this case we'll have to `cd` to an actual directory that has a file which path is provided by `directory`. What concerns me a bit is that the action will have to act differently depending on what is pointed by `directory`.

Another complication is that the action has to write its output to a log. Currently that is `kubelinter.log`, in the future that'll be a sarif file. I think, it is better that this log is stored in the root of the repo, not in the scanned sub-directory.
It is not hard to solve this by constructing the absolute path with `github.workspace` though.

However, I think that `kube-linter` needs to be adjusted in this case. I think it should look for config file in the provided location, then look for it in the parent directory of the location, then parent of a parent and so on until it either finds the config file or cannot go up to the parent directory any more (because it reached root or because it does not have permissions to go up).

With the current logic of using config file in the _current_ directory, scan results for the same files will depend on where you are when running `kube-linter`. I imagine this can be surprising when you are currently in one tree with its config and you want to scan another tree with a different config. The first config will be used and the one sitting with the scanned files will be ignored.

What do you think?

Reply:
This makes a lot of sense, and is something I'd tentatively put down in https://github.com/stackrox/kube-linter/issues/49. I agree that we should adjust `kube-linter` first and then we can get back to the action.